test_players/load_engine.js

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
Can we reuse/extend engineChange.js instead of writing an entirely new file?

Reply:
to consider in the next sprint